ksql-execution/src/main/java/io/confluent/ksql/execution/ddl/commands/CreateTableCommand.java

```diff
@@ -26,22 +27,22 @@
 
 @Immutable
 public class CreateTableCommand extends CreateSourceCommand {
-
   public CreateTableCommand(
-      final String sqlExpression,
-      final SourceName sourceName,
-      final LogicalSchema schema,
-      final Optional<ColumnName> keyField,
-      final TimestampExtractionPolicy timestampExtractionPolicy,
-      final Set<SerdeOption> serdeOptions,
-      final KsqlTopic ksqlTopic
+      @JsonProperty(value = "sqlExpression", required = true) final String sqlExpression,
+      @JsonProperty(value = "sourceName", required = true) final SourceName sourceName,
+      @JsonProperty(value = "schema", required = true) final LogicalSchema schema,
+      @JsonProperty(value = "keyField", required = true) final Optional<ColumnName> keyField,
+      @JsonProperty(value = "timestampExtractionPolicy", required = true)
```

Comment:
as above: can we keep the names shorter?

```suggestion
      @JsonProperty(value = "timestampExtractor", required = true)
```